requirements/testing.txt

```diff
@@ -35,9 +35,9 @@ pyhive[hive,presto]==0.6.3  # via -r requirements/development.in, -r requirement
 pylint==2.6.0             # via -r requirements/testing.in
 pytest-cov==2.10.1        # via -r requirements/testing.in
 pytest==6.1.2             # via -r requirements/testing.in, pytest-cov
-redis==3.5.3              # via -r requirements/testing.in
```

Comment:
Redis is currently required for running tests.

Reply:
@robdiciuccio this will be inherited from `base.in`.